### ISSUE TITLE
Network Errors Logged On Page Load

### ISSUE THREAD

**Issue by osscontributor** [grails/grails-forge#283]
<img width="1350" alt="Screenshot 2024-02-13 at 9 15 03 AM" src="https://github.com/grails/grails-forge/assets/15482/55fd629d-8509-413c-8e2a-230e2c631a7c">

When I load the page in the browser I am seeing errors sending requests to https://prev.grails.org/versions  and  https://next.grails.org/versions coming out of MicronautStartSDK.js.

FYI: https://github.com/micronaut-projects/micronaut-starter-ui/issues/102 looks like it might be tangentially related, but probably not the same issue.

**Comment by puneetbehl** [comment]
This has no side-effects. Currently, there are no previous or next version of Grails framework supported with the current CLI that's why these requests are failing. 

Once, we have some milestone available for Grails 7, that would point to next.grails.org. And, similarly once the GA for 7 is out. The Grails 6 will point to prev.grails.org. So, I believe the DNS will be:

https://prev.grails.org - Grails 6
https://latest.grails.org - Grails 7
https://next.grails.org - Grails 8 Milestone or RC.

**Comment by puneetbehl** [comment]
No, but I think it is moved here as part of initial setup. We can plan to update the codebase to not make these calls for now.

**Comment by osscontributor** [comment]
> We can plan to update the codebase to not make these calls for now.

Sounds good.  Thank you!
